cg/services/slurm_service/slurm_cli_service.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+from cg.apps.slurm.slurm_api import SlurmAPI
+from cg.models.slurm.sbatch import Sbatch
+from cg.services.slurm_service.slurm_service import SlurmService
+
+
+class SlurmCLIService(SlurmService):
+
+    def __init__(self) -> None:
+        self.client = SlurmAPI()
+        self.account = ""
+        self.log_dir = ""
+        self.email = ""
```

Comment:
Should I use the `data_delivery` config or create a new one? If I use data delivery:
- account: development/production
- email:  clinical-logwatch@scilifelab.se
- log_dir: rsync

Then, the output directory for the sbatch script and logs would become
`/home/proj/production/rsync/<upload job name>/`. Is that a good idea?